tee-worker/litentry/pallets/identity-management/src/tests.rs

```diff
@@ -121,6 +121,29 @@ fn link_identity_with_wrong_network_fails() {
 	});
 }
 
+#[test]
+fn link_identity_with_empty_network_works() {
+	new_test_ext(true).execute_with(|| {
+		let web3networks: Vec<Web3Network> = vec![];
+		let who: Identity = BOB.into();
+		assert_ok!(IMT::link_identity(
+			RuntimeOrigin::signed(ALICE),
+			who.clone(),
+			alice_evm_identity(),
+			web3networks,
+		));
+		assert_eq!(
+			IMT::id_graphs(who.clone(), alice_evm_identity()).unwrap(),
+			IdentityContext {
+				link_block: 1,
+				web3networks: vec![Web3Network::Ethereum, Web3Network::Bsc].try_into().unwrap(),
```

Comment:
can we do something like this so it doesn't break on new evm networks? 
```suggestion
				web3networks: all_evm_web3networks().try_into().unwrap(),
```

Reply:
Actually I wrote this intentionally since we used `all_evm_web3networks` in the impl, let me change it back